packages/yourloops/components/routes.tsx

```diff
@@ -57,7 +60,9 @@ export const PrivateRoute = (props: RouteProps): JSX.Element => {
   return isLoggedIn() ? (
     <ThemeProvider theme={mainTheme}>
       <CssBaseline />
-      <Route {...props} />
+      <SnackbarContextProvider context={DefaultSnackbarContext}>
```

Comment:
i will have put it in app.ts to avoid twice declarations, however we could leave it this way

Reply:
It was my original idea. But the Theme is different, and all the colors in the rest of the UI change when an alert is displayed.
So to make it quick I duplicate this.

Reply:
oki it is fine for me